worker/orca_grader/__main__.py

```diff
@@ -160,8 +161,7 @@ def run_grading_job(grading_job: GradingJobJSON, no_container: bool,
 # TODO: Would it be more useful to return the result of the job here?
 def handle_grading_job(grading_job: GradingJobJSON, image_name: str | None = None,
                        container_cmd: List[str] | None = None) -> None:
-    with tempfile.NamedTemporaryFile(mode="w", suffix='.json',
-                                     delete_on_close=False) as temp_job_file:
+    with tempfile.NamedTemporaryFile(mode="w", suffix='.json') as temp_job_file:
```

Comment:
This change worries me -- I'm not convinced that we'll be able to `docker cp` the tempfile if we do this.  Keep in mind that we still have delete=True, so the file still will be deleted at the end of the with-statement...

Reply:
Fair point, realized I had misinterpreted the doc for `delete` vs `delete_on_close`; fixed now!